.github/styles/Vocab/Docs/accept.txt

```diff
@@ -1,24 +1,67 @@
-D2iQ
-COPS
-Mesos
+addon
+Airflow
+Alertmanager
+Ansible
+autoscal(s|ed|ing)
```

Comment:
I don't think autoscal is a word

Reply:
Pushed https://github.com/mesosphere/dcos-docs-site/pull/3214/commits/8068e2bfa9ed2e3b6d38baa1597e764942857d45 to fix.